src/__tests__/client.test.ts

```diff
@@ -0,0 +1,22 @@
+import Client from '../client';
+import { PollingProcessor } from '../polling';
+
+jest.mock('../openapi/api');
+
+it('check close client', async () => {
```

Comment:
Totally personal preference, but a test's name should read like a requirement. If you're using `it`, then it makes sense that the test name should start with `should` and describe the expected outcome.

In this case, maybe something like...
```suggestion
it('should close the client when the close method is called', async () => {
```